### PR TITLE
Update meta tags for SEO

### PR DESCRIPTION
**Pull request by beth-panx**

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -37,6 +37,9 @@
   />
   <meta name="twitter:image" content="https://microsoft.github.io/win-student-devs/30DaysOfPWA/_media/social.jpg" />
 
+  <link href="/feed.xml" rel="alternate" type="application/xml" data-title="30 Days of PWA feed">
```
Comment (tomayac) [inline docs/index.html:40]:
```suggestion
  <link href="/feed.xml" rel="alternate" type="application/atom+xml" data-title="30 Days of PWA feed">
```

File: docs/index.html [hunk 2]
```diff
@@ -37,6 +37,9 @@
...
+  <link rel="manifest" href="manifest.json">
```
Comment (tomayac) [inline docs/index.html:41]:
```suggestion
  <link rel="manifest" href="manifest.json" />
```

**Review by tomayac — COMMENTED** [review]
Thanks for working on this. As you wrote, user-visible links would be great, too.

File: docs/index.html [hunk 3]
```diff
@@ -37,6 +37,9 @@
   />
   <meta name="twitter:image" content="https://microsoft.github.io/win-student-devs/30DaysOfPWA/_media/social.jpg" />
 
+  <link href="/feed.xml" rel="alternate" type="application/atom+xml" data-title="30 Days of PWA feed" />
```
Comment (beth-panx) [inline docs/index.html:40]:
```suggestion
  <link href="/feed.xml" rel="alternate" type="application/atom+xml" title="30 Days of PWA feed" />
```

**Review by zateutsch — APPROVED** [review]

**Pull request merged**